homework-selects/sql_questions.sql

```diff
@@ -4,54 +4,281 @@ SELECT first_name, last_name, salary, job_title
 FROM employees
```

Comment:
for future, please follow this rule!
https://towardsdatascience.com/10-best-practices-to-write-readable-and-maintainable-sql-code-427f6bb98208

"1. Use Uppercase for the Keywords
Let’s start with a basic: use uppercase for the [SQL keywords](https://www.w3schools.com/sql/sql_ref_keywords.asp), and lowercase for your tables and columns. It’s also a good practice to use uppercase for the SQL functions (FIRST_VALUE(), DATE_TRUNC(), etc) even though it’s more debatable."

Reply:
sure